include/config.h

```diff
@@ -7,6 +7,7 @@
 #ifdef WIN32
 # ifdef __MINGW32__
 #  include <winsock2.h>
+#  include <windows.h>
```

Comment:
I assume this would not hurt even if defined immediately under WIN32. It's just that under VS it isn't really needed.